docs/workforce-management-framework/upstream-1/working-with-passportauth.adoc

```diff
@@ -1,25 +1,17 @@
 include::topics/shared/attributes.adoc[]
 
-//':chapter:' is a vital parameter. See: http://asciidoctor.org/docs/user-manual/#include-multiple
-:chapter: working-with-passportjs
+//':context:' is a vital parameter. See: http://asciidoctor.org/docs/user-manual/#include-multiple
+:context: working-with-passportauth
 
 = Working With PassportAuth
 
 //Concepts
 include::topics/con-passportauth-module.adoc[leveloffset=+1]
 
 //Procedures
-include::topics/pro-working-with-passportauth.adoc[leveloffset=+1]
+include::topics/pro-passportauth-pointing-to-a-datasource.adoc[leveloffset=+1]
 
+//Reference Material
 == PassportAuth Reference Material
-The security aspects of *PassportAuth* module are described in the Security Strategy and cover areas such as:
```

Comment:
this content is now part of "include::topics/con-passportauth-securitystrategy.adoc[leveloffset=+1]" (see new change -->) and imported